### ISSUE TITLE
error in "explain" function with multilabel classification

### ISSUE THREAD

**Issue by hebatallah** [amaiya/ktrain#55]
An error when I try to use the "explain" function with multilabel classification I get this error

  File "C:/Users/HeMohamed/Documents/Python Scripts/PhD-TagPred/ktrain.py", line 239, in <module>
    predictor.explain(doc)

  File "C:\Users\HeMohamed\Anaconda3\envs\tensorflow_cpu\lib\site-packages\ktrain\text\predictor.py", line 105, in explain
    _ = te.fit(doc, self.predict_proba)

  File "C:\Users\HeMohamed\Anaconda3\envs\tensorflow_cpu\lib\site-packages\eli5\lime\lime.py", line 265, in fit
    random_state=self.rng_

  File "C:\Users\HeMohamed\Anaconda3\envs\tensorflow_cpu\lib\site-packages\eli5\lime\lime.py", line 367, in _train_local_classifier
    random_state=rng)

  File "C:\Users\HeMohamed\Anaconda3\envs\tensorflow_cpu\lib\site-packages\eli5\lime\utils.py", line 32, in fit_proba
    random_state=random_state,

  File "C:\Users\HeMohamed\Anaconda3\envs\tensorflow_cpu\lib\site-packages\eli5\lime\utils.py", line 111, in expanded_X_y_sample_weights
    sample_weight

  File "C:\Users\HeMohamed\Anaconda3\envs\tensorflow_cpu\lib\site-packages\eli5\lime\utils.py", line 140, in expand_dataset
    n_classes = y_proba.shape[1]

AttributeError: 'list' object has no attribute 'shape'

**Comment by amaiya** [comment]
*ktrain* uses the `eli5` and `lime` libraries to generate explanations.  It doesn't look `eli5` supports multilabel classification as shown in this [open eli5 issue](https://github.com/TeamHG-Memex/eli5/issues/337) (although there is a workaround mentioned in that thread that may or may not make sense to do).

That said, the current error being printed (i.e., list object has no attribute shape) is the result of a bug.  The correct error should be:  `ValueError: probabilities do not sum to 1`.  The correct error will be printed in next release.
